src/locale/locales/fr/messages.po

```diff
@@ -390,7 +382,7 @@ msgstr "Contenu pour adultes"
 
 #: src/screens/Moderation/index.tsx:356
 msgid "Adult content can only be enabled via the Web at <0>bsky.app</0>."
-msgstr ""
+msgstr "Le contenu pour adultes ne peut être activé que via le Web à <0>bsky.app</0>."
```

Comment:
We use either `à l'adresse` or `sur` (like English's `on`) for websites references like URLs. The first being longer and less fluid imho, let's use the latter.

```suggestion
msgstr "Le contenu pour adultes ne peut être activé que via le Web sur <0>bsky.app</0>."
```

Reply:
Ah got it 👍